eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -9,49 +9,49 @@ parameters:
   BuildDocs: true
   TestMatrix:
     Linux_Python27:
-      OSName: 'Linux'
+      OSVmName: 'Linux'
       OSVmImage: 'ubuntu-18.04'
       PythonVersion: '2.7'
       CoverageArg: ''
       RunForPR: true
     Linux_Python35:
-      OSName: 'Linux'
+      OSVmName: 'Linux'
```

Comment:
We are just removing this completely for all the matrix entries.

Reply:
We use it when publishing the test run title, so I'm going to keep the parameter around (just renamed). 

I should just be able to remove the parameter being passed to the common yml yes?

Reply:
It will be set within the script. So I think you don't need the variable at all, so long as the verify-agent-os script runs a s the first step. it is set [here](https://github.com/Azure/azure-sdk-for-python/blob/9d242098da666311c76ae82de07bf5161c7929dd/eng/common/pipelines/templates/steps/verify-agent-os.yml#L36)